src/validateEmail.test.js

```diff
@@ -17,4 +17,147 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   // write more tests here
+
+  it('should return "false" non-english letters', () => {
+    expect(validateEmail('bąk@doom.com'))
+      .toBe(false);
+  });
+
+  it('should return "true" for digits in email', () => {
+    expect(validateEmail('email123@mail123.com123'))
+      .toBeTruthy();
+  });
+
+  it('should return "true" for only digits email', () => {
+    expect(validateEmail('12424@342.234'))
+      .toBeTruthy();
+  });
+
+  it('should return "true" for "_" in email', () => {
+    expect(validateEmail('e_mail@test.com'))
+      .toBeTruthy();
+  });
+
+  it('should return "true" for "-" in email', () => {
+    expect(validateEmail('e-mail@test.com'))
+      .toBeTruthy();
+  });
+
+  it('should return "false" if "." or "," is on begening of email', () => {
+    const mail1 = '.email@test.com.';
+    const mail2 = ',email@test.com.';
+
+    expect(validateEmail(mail1))
+      .toBe(false);
+
+    expect(validateEmail(mail2))
+      .toBe(false);
+  });
+
+  it('should return "false" if ",," or ".." is in email', () => {
+    const mail1 = 'ema,,il@test.com.';
+    const mail2 = 'em..ail@test.com.';
+
+    expect(validateEmail(mail1))
+      .toBe(false);
+
+    expect(validateEmail(mail2))
+      .toBe(false);
+  });
+
+  it('should return "false" if "@" is not in email', () => {
+    expect(validateEmail('email2test.com'))
+      .toBe(false);
+  });
+
+  it(`should return false if "! $ % & ' * + / = ? ^ { | } ~" is in email`
```

Comment:
The description of this test case is misleading. Some of the characters listed (e.g., '!', '$', '%', etc.) are actually valid in the local part of an email address. Consider revising the test case to accurately reflect which characters are invalid according to your validation logic.